src/autopas/utils/ArrayMath.h

```diff
@@ -186,6 +186,40 @@ template <class T, std::size_t SIZE>
   return result;
 }
 
+/**
+ * Divides two array's element-wise and returns the result.
```

Comment:
```suggestion
 * Divides an array element-wise by a given scalar and returns the result.
```
Check this sentence also for the other implementations.